odoo/tools/misc.py

```diff
@@ -1363,6 +1364,28 @@ def format_duration(value):
     return '%02d:%02d' % (hours, minutes)
 
 
+def format_float(env, value, precision_rounding=None, uom=None, lang_code=False):
+    if precision_rounding is not None:
+        rounding = precision_rounding
+    elif uom is not None:
```

Comment:
This is a bit strange as tools should be as model independent as possible. I would simply allow a rounding to format_float, and caller has to either give it manually or either from an uom.

I know some other tools use records (like currency), but currency is defined in base while uom in uom.

Reply:
good idea for `uom_name`